recipe/meta.yaml

```diff
@@ -12,14 +12,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   skip: True  # [win or py<=27 or py>=37]
```

Comment:
- release python version constrain
- add skip for OSX

Reply:
This change has been made.